cmd/describe.go

```
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/redhat-developer/ocdev/pkg/application"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/redhat-developer/ocdev/pkg/occlient"
+	"github.com/redhat-developer/ocdev/pkg/project"
+	"github.com/redhat-developer/ocdev/pkg/storage"
+	"github.com/redhat-developer/ocdev/pkg/url"
+	"github.com/spf13/cobra"
+)
+
+var componentDescribeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: "describe the given application",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+		currentApplication, _ := application.GetCurrent(client)
+		currentProject := project.GetCurrent(client)
+		var cmpnt string
+		if len(args) == 0 {
+			cmpnt, _ = component.GetCurrent(client)
```

ignoring the error :)

---

omg, sorry about this :(